examples/2-using-subscriptions/yarn.lock

```diff
@@ -4869,12 +4816,10 @@ wide-align@^1.1.0:
   dependencies:
     string-width "^1.0.2 || 2"
 
-wonka@^2.0.1:
-  version "2.0.1"
-  resolved "https://registry.yarnpkg.com/wonka/-/wonka-2.0.1.tgz#3769f9b050683edf2280e04d2b01afbdfe5fd9d4"
-  integrity sha512-s0nGPVbbA9a0FhUarNJino4i1J1UyQ5rGA1MmA65U/+xUXmBQEw4cCKceRXkDhgL8ZJ2HVhvxthHFZZTf02gBA==
-  dependencies:
-    bs-rebel "^0.2.3"
+wonka@^3.0.0:
```

Comment:
Looks like our lock file was out of sync with our package.json

Reply:
This was just for the example project, so not too concerned about it for the time being. I verified the example starts up as expected 👍